tools/src/main/java/it/eng/tools/rest/api/ApplicationPropertiesAPIController.java

```diff
@@ -52,15 +46,14 @@ public ApplicationPropertiesAPIController(ApplicationPropertiesService service,
 	 * @return List of properties
 	 */
 	@GetMapping(path = "/")
-	public ResponseEntity<List<JsonNode>> getProperties(@RequestParam(required = false) String key_prefix) {
+	public ResponseEntity<GenericApiResponse<List<ApplicationProperty>>> getProperties(@RequestParam(required = false) String key_prefix) {

```

Comment:
Nice one for changing to GenericApiResponse! As for the data type, maybe that one should be left to List<JsonNode> or JsonNode depending on the logic, since all APIs are like that